docs/guides/react-components/saas-prerequisites.md

```diff
@@ -0,0 +1,26 @@
+---
+---
+
+## Sign Up
+
+Visit [camunda.io/signup](https://signup.camunda.com/accounts?utm_source=docs.camunda.io&utm_medium=referral) to view the sign up screen:
+
+![signup](./../img/signup.png)
+
+### Create an account
+
+Fill out the form and click **Create account**, or sign up using the social sign up buttons like Google or GitHub.
```

Comment:
When I visit the link above I don't see the social sign up buttons -- do we still support this?

Reply:
Good spot. Looks like we don't support this for account creation (just login) and the screenshot is out of date.

Reply:
I'll fix it in this PR, but can you create a new PR for us to update the text and screenshot?